src/gssproxy.c

```diff
@@ -114,6 +115,11 @@ int main(int argc, const char *argv[])
     }
 
     gpctx = calloc(1, sizeof(struct gssproxy_ctx));
+    if (!gpctx) {
+        ret = ENOMEM;
```

Comment:
Please change this to EXIT_FAILURE like for all other error sites in the main() function.

Reply:
done